verticapy/core/vdataframe/_aggregate.py

```diff
@@ -1217,6 +1269,56 @@ def aad(
         -------
         TableSample
             result.
+
+        Examples
+        --------
+        For this example, we will use the following dataset:
+
+        .. code-block:: python
+
+            import verticapy as vp
+
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+        
+        Now, let's calculate the average absolute deviation for 
+        specific columns.
+
+        .. code-block:: python
+
+            data.aad(
+                columns = ["x", "y", "z"],
+            )
+
+        .. ipython:: python
+            :suppress:
+
+            import verticapy as vp
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+            result = data.aad(
+                columns = ["x", "y", "z"],
+            )
+            html_file = open("figures/core_vDataFrame_vDFAgg_aad_table.html", "w")
+            html_file.write(result._repr_html_())
+            html_file.close()
+
+        .. raw:: html
+            :file: SPHINX_DIRECTORY/figures/core_vDataFrame_vDFAgg_aad_table.html
+
+        .. note:: All the calculations are pushed to the database.
+
+        .. hint:: For more precise control, please refer to the `aggregate` method.
```

Comment:
we can instead use a reference for the module: :py:mod:`verticapy.vDataColumn.aggregate`

Reply:
They can click on the see also below